finetuner/tuner/pytorch/__init__.py

```diff
@@ -197,6 +195,10 @@ def fit(
 
             self._trigger_callbacks('on_train_epoch_begin')
             self._train(train_dl)
+
+            if self._scheduler_step == 'epoch' and self._scheduler is not None:
```

Comment:
```suggestion
            if self._scheduler and self._scheduler_step == 'epoch' :
```
if you want to check it exist in an if, should be checked first.

Reply:
I don't see what improvement this brings?

Reply:
okay i thought you're checking scheduler, did not notice the second is `_scheduler_step`